products/acsf/features/php-administrator.feature

```diff
@@ -116,6 +123,7 @@ Feature: PHP
     And the cache has been cleared
     When I am on "admin/structure/flags/import"
     And I should see "This form is disabled"
+    And the response status code should be 403
```

Comment:
I'm getting 200 on this page, in testing and when I visit it manually.

Reply:
If it's returning "This form is disabled", I shouldn't be checking for 403. I'm not sure how this passed on local. Let me revisit.

Reply:
Removed in bd06adf